test/Graphics/ddx_fine.test

```diff
@@ -0,0 +1,97 @@
+#--- vertex.hlsl
+struct PSInput
+{
+    float4 position : SV_POSITION;
+};
+
+PSInput main(float4 position : POSITION)
+{
+    PSInput result;
+
+    result.position = position;
+
+    return result;
+}
+
+
+#--- pixel.hlsl
+
+// Offset into the image of our circle
+#define offset 128.0
```

Comment:
Arguably better to either name this `OFFSET` following the usual preprocessor macro naming conventions or make it a `static const float` instead of using a define.